CI.md

```diff
@@ -0,0 +1,475 @@
+Then the most basic build script can be specified as follows:
+
+```yml
+# This image may be changed when needed,
+# but the recommendation is to keep it on
+# Visual Studio 2019.
+image: Visual Studio 2019
+
+#---------------------------------#
+#  Build Script                   #
+#---------------------------------#
+build_script:
+  - ps: .\build.ps1 -Target CI
+
+# Tests
+test: off
+
+#---------------------------------#
+#        Branches to build        #
+#---------------------------------#
+branches:
+  # Whitelist
+  only:
+    - develop
+    - master
+    - /release/.*/
+    - /hotfix/.*/
+
+#---------------------------------#
+#  Build Cache                    #
+#---------------------------------#
+cache:
+  # Update this cache variable to match
+  # your current setup.
+  - "tools -> recipe.cake,tools/packages.config"
+
+#---------------------------------#
+#  Environment Variables          #
+#---------------------------------#
+
+environment:
+  # We set MYGET_SOURCE to an empty variable due to known issues when pushing
+  # packages to MyGet
+  MYGET_SOURCE:
+```
+
+**WARNING: At this time it is not recommended to add a matrix to build on
+multiple platforms on AppVeyor due to the increased build time necessary.**
+
+### Using Cake.Recipe 1.x with AppVeyor
+
+Configuring AppVeyor to build projects targeting Cake.Recipe 1.x is very similar
+to how to use Cake.Recipe 2.x documentation.
+The main change between the build script on AppVeyor is mainly to change the
+build target from `CI` to `AppVeyor`.
+See the below build script as an example of this.
+
+```yml
+image: Visual Studio 2017
+#---------------------------------#
+#  Build Script                   #
+#---------------------------------#
+build_script:
+  - ps: .\build.ps1 -Target AppVeyor
+
+# Tests
+test: off
+
+#---------------------------------#
+#        Branches to build        #
+#---------------------------------#
+branches:
+  # Whitelist
+  only:
+    - develop
+    - master
+    - /release/.*/
+    - /hotfix/.*/
+
+#---------------------------------#
+#  Build Cache                    #
+#---------------------------------#
+cache:
+  # Update this cache variable to match
+  # your current setup.
+  - "tools -> recipe.cake,tools/packages.config"
+
+#---------------------------------#
+#  Environment Variables          #
+#---------------------------------#
+
+environment:
+  # We set MYGET_SOURCE to an empty variable due to known issues when pushing
+  # packages to MyGet
+  MYGET_SOURCE:
+```
+
+**WARNING: At this time it is not recommended to add a matrix to build on
+multiple platforms on AppVeyor due to the increased build time necessary.**
+
+#### Additional notes
+
+If you do not plan to use AppVeyor as the preferred build platform,
+it may not be possible to use Cake.Recipe 1.x. Please migrate to the latest 2.x
+pre-release instead.
+
+## Using Azure Pipelines
+
+There is currently no documentation on how to set up Cake Contrib addins for
```

Comment:
I think that @pascalberger might be able to help here, as I know he runs some builds over there with Cake.Recipe.

Reply:
That was my thought as well, hence why I kept the header for Azure Pipelines :laughing:

Reply:
@gep13 @AdmiringWorm Yes, I can add something.

Reply:
@pascalberger woot! Thank you!